include/phasar/PhasarLLVM/Plugins/Interfaces/ControlFlow/ICFGPlugin.h

```diff
@@ -33,14 +33,14 @@ class ICFGPlugin
   const std::vector<std::string> EntryPoints;
 
 public:
-  ICFGPlugin(ProjectIRDB &IRDB, const std::vector<std::string> EntryPoints)
-      : IRDB(IRDB), EntryPoints(move(EntryPoints)) {}
+  ICFGPlugin(ProjectIRDB &IRDB, const std::vector<std::string> &EntryPoints)
+      : IRDB(IRDB), EntryPoints(EntryPoints) {}
```

Comment:
Why would we not move the EntryPoints here?

```suggestion
  ICFGPlugin(ProjectIRDB &IRDB, std::vector<std::string> EntryPoints)
      : IRDB(IRDB), EntryPoints(std::move(EntryPoints)) {}
```